my-webforum/src/CommentData.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+export const comments = [
+  {
+    profileImg: 'https://images.unsplash.com/photo-1609992556706-14a7056ea745?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1287&q=80',
+    username: 'ScrungeCat',
+    comment: 'My favorite types of cats are slightly weird looking ones!'
+  },
+  {
+    profileImg: 'https://images.unsplash.com/photo-1615751072497-5f5169febe17?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1335&q=80',
+    username: 'ChewToy',
+    comment: 'I don\'t like cats at all.'
+  },
+  {
+    profileImg: 'https://images.unsplash.com/photo-1563482776068-4dac10f9373d?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1170&q=80',
+    username: 'BuryHeadInSand',
+    comment: 'Wild ostriches make the best pets.'
+  }
+]
+
+const Comment = () => {
```

Comment:
Why does this component not have a prop? Why is it "empty"? Why was it not added in a separate file? It doesn't appear to be used, why was it added?

Reply:
It was a mistake sir, it has been updated now!